### PR TITLE
feat(icons): Generate icons and updates to svg structures

### PR DESCRIPTION
**Pull request by mikeyrayvon**
Generates IconCamera and IconFolderArrowRight

### CODE REVIEW

File: src/foundation/Icon/Generated/IconBell12.tsx [hunk 1]
```diff
@@ -11,21 +11,12 @@ function IconBell12(props: GeneratedIconProps): React.ReactElement<GeneratedIcon
             name="IconBell12"
             {...props}
         >
-            <g fill="none">
```
Comment (imoutaharik) [inline src/foundation/Icon/Generated/IconBell12.tsx:14]: Mmmm, Why a lot of size 12 icons got updated? That `<g fill="none"` may cause an issue in Clarify with a possibility of the icons going blank.

Comment (mikeyrayvon) [reply src/foundation/Icon/Generated/IconBell12.tsx:14]: No clue! I just pulled the branch and ran the generate icons command...

File: src/foundation/Icon/Generated/IconBell12.tsx [hunk 2]
```diff
@@ -11,21 +11,12 @@ function IconBell12(props: GeneratedIconProps): React.ReactElement<GeneratedIcon
...
-                <g clipPath="url(#prefix__a)">
-                    <path
-                        fill="currentColor"
```
Comment (mikeyrayvon) [inline src/foundation/Icon/Generated/IconBell12.tsx:17]: But it looks like the fill on the path was (and will still be) `currentColor`. At least in this case

File: src/foundation/Icon/Generated/IconBell12.tsx [hunk 3]
```diff
@@ -11,21 +11,12 @@ function IconBell12(props: GeneratedIconProps): React.ReactElement<GeneratedIcon
...
-                    />
-                </g>
-                <defs>
-                    <clipPath id="prefix__a">
-                        <path fill="#fff" d="M0 0h12v12H0z" />
-                    </clipPath>
-                </defs>
-            </g>
+            <path
+                fill="currentColor"
```
Comment (mikeyrayvon) [inline src/foundation/Icon/Generated/IconBell12.tsx:15]: See here

**Pull request closed without merging**